### ISSUE TITLE
False positive in Lint/FormatParameterMismatch for argument with splat operator

### ISSUE THREAD

**Issue by jonas054** [bbatsov/rubocop#2232]
Extracted from #2123. Reported by @dreyks.

What about the opposite situation?

```ruby
v = [1, 2]
format('%d%d', *v)
```

```
W: Number arguments (1) to format mismatches expected fields (2).
```

this should probably be ignored too.

**Comment by dreyks** [comment]
Ok seems like this does the trick 
#2239
